invenio_app_ils/permissions.py

```diff
@@ -55,7 +55,5 @@ def views_permissions_factory(action):
     """Default ILS views permissions factory."""
     if action == "circulation-loan-request":
         return authenticated_user_permission()
-    elif action == "ils-backoffice-view":
```

Comment:
The view is now unprotected, there is an issue open for this: https://github.com/inveniosoftware/invenio-app-ils/issues/90